index.html

```diff
@@ -4,7 +4,154 @@
+        <img
+          id="header-img"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+          alt="original trombones logo"
+        />
+        <ul>
+          <li>
+            <a class="nav-link" href="#features">Features</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#how_its_works">How It Works</a>
+          </li>
+          <li>
+            <a class="nav-link" href="#pricing">pricing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div>
+      <section id="hero">
+        <h2 class="wick">Handcrafted, home-made masterpieces</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input
+            name="email"
+            id="email"
+            class="style"
+            type="email"
+            placeholder="Enter your email address"
+            required
+          />
+          <input id="submit" class="change" type="submit" value="Get Started" />
+        </form>
+      </section>
+    </div>
+    <div class="undone">
+      <img src="images/flexible_tail.png" class="reuse" alt="flexible_tail.png" />
+      <div id="features">
+        <h2 class="Edit">Features</h2>
+        <p class="edit">
+          A broad scope that covers all weapons systems that select and engage targets on the basis
+          of sensor inputs—that is, systems in which the object to be attacked is determined by
+          sensor processing, not by humans
+        </p>
+      </div>
+    </div>
+    <div class="undone">
+      <img src="images/click_virtual.png" class="reuse" alt="click virtual" />
+      <div id="how_its_works">
+        <h2 class="Edit">How its Works</h2>
+        <p class="edit">
+          A broad scope that covers all weapons systems that select and engage targets on the basis
+          of sensor inputs—that is, systems in which the object to be attacked is determined by
+          sensor processing, not by humans
+        </p>
+      </div>
+    </div>
+    <div class="undone">
+      <img
+        src="images/standard_quality_control.jpg"
+        class="reuse"
+        alt="standard_quality_control.png"
+      />
+      <div id="pricing">
+        <h2 class="Edit">Prising</h2>
+        <p class="edit">
+          A broad scope that covers all weapons systems that select and engage targets on the basis
+          of sensor inputs—that is, systems in which the object to be attacked is determined by
+          sensor processing, not by humans
+        </p>
+      </div>
+    </div>
+
+    <video
+      id="video"
+      class="stylesheet"
+      src="https://www.youtube.com/embed/tgbNymZ7vqY?controls=0"
+    ></video>
+    <iframe width="420" height="315" src="https://www.youtube.com/embed/tgbNymZ7vqY?controls=0">
+    </iframe>
+
+    <section class="changes">
+      <div class="product">
+        <h4 class="level">Bass Trombone</h4>
+        <h2 class="size">$900</h2>
```

Comment:
It's best to arrange `h` tags in a hierarchy for better accessibility.
e.g. 
`h1`
`h2`
`h3`
... `h6`

Reply:
Pls I don't get this pls accept it like that i will try to adjust next time